app/src/main/java/sdp/moneyrun/player/LocalPlayer.java

```diff
@@ -8,40 +8,46 @@
 /**
  * We store here all the attributes of a player that do not need no be pushed to the database
  */
-public class LocalPlayer  implements Serializable {
+public class LocalPlayer implements Serializable {
 
-    private ArrayList<Coin> lostCoins;
+    private final ArrayList<Coin> lostCoins;
     private ArrayList<Coin> locallyAvailableCoins;
```

Comment:
Is this attribute only useful for offline mode? That is to have some coins that are not from the database only when online database is not available?

Reply:
I am not sure what you mean, but yes, it will only be used locally.

Reply:
Sorry for the confusion, I will rephrase, what is the difference between `locallyAvailableCoins` and the coins stored on the database, since the coins on the database should be updated once someone picks up the coin. Here we have the attribute `lostCoins` that indicates that some coins are not available for some player but why do we need an attribute for the available ones, the only reason I see is when database is not available(like in offline mode) to have a local copy.

Reply:
The idea is to have a list of coins that are available to the player locally, so basically all the database available coins - lost coins.
This avoids having to check every time if the coin is in lost coins while looping through the database available coins.

Reply:
Okay, yes I agree